README.md

```diff
@@ -1 +1,153 @@
+    # run phase
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run sysbench oltp_insert --threads=10 --time=30 --table_size=10000 --tables=1 --warmup-time=0 --range_key_partitioning=false --range_selects=true --thread-init-timeout=30 --serial_cache_size=1000 --db-driver=pgsql --pgsql-db=yugabyte --pgsql-port=5433 --pgsql-user=yugabyte --pgsql-host=<node1-ip,node2-ip,node3-ip> run
+    ```
+  - ##### Running ycsb 
+     Info about ycsb specific parameters can be found [here](https://github.com/yugabyte/ycsb).
+    ```shell
+    # load
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run ycsb load basic -P workloads/workloada
+     
+    # run
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run ycsb run basic -P workloads/workloada
+    ```
+    To modify the workload related properties in YCSB -[db.properties](https://github.com/yugabyte/YCSB/blob/master/yugabyteSQL/db.properties), you can mount the file as a volume.
+    ```shell
+    # download the file
+    wget https://raw.githubusercontent.com/yugabyte/YCSB/master/yugabyteSQL/db.properties
+     
+    # modify the db.properties in any editor and run the benchmark
+    docker run -v $(pwd)/db.properties:/home/centos/code/YCSB/yugabyteSQL/db.properties \
+    --name ybbench --rm -it yugabytedb/ybbench:latest ./run ycsb load basic -P workloads/workloada
+    ```    
+  
+    Alternatively, you can run ybbench in [interactive mode](#ybbench-in-interactive-mode)
+  
+  - ##### Running yb-sample-apps
+     Info about yb-sample-apps parameters can be found [here](https://github.com/yugabyte/yb-sample-apps).
+    ```shell
+    docker run --name ybbench --rm -it yugabytedb/ybbench:latest ./run yb-sample-apps --workload CassandraKeyValue --value_size 16 --num_unique_keys 1000000 --num_threads_read 0 --num_threads_write 256 --nodes <node1-ip>:9042,<node2-ip>:9042,<node3-ip>:9042 --use_ascii_values create_table_name PerfTest_0 --output_json_metrics 
+    ```
+## Build from source code
+You can build the docker image from the specific branches of respective benchmark repositories.
+1. Clone Repository
+    ```shell
+    git clone https://github.com/yugabyte/ybbench.git
+    ```
+2. Run docker build
+    ```shell
+    cd ybbench
+    # master branches is used by default if not specified
+    docker build -t ybbench:0.1 .
+    ```
+    Docker Build with Specific Branch
+    ```shell
+    docker build -t ybbench:0.1 --build-arg tpcc_branch=<my-branch> \
+    --build-arg sysbench_branch=<my-branch> --build-arg sample_apps_branch=<my-branch> \
+    --build-arg ycsb_branch=<my-branch> .
+    ```
+
+## ybbench in interactive mode
```

Comment:
can you move this section above the "Build from source" section

Reply:
done